bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -106,10 +107,12 @@ public final void onSuccess(Object message) {
       try {
         if (message instanceof FlatRow) {
           result = Adapters.FLAT_ROW_ADAPTER.adaptResponse((FlatRow) message);
-        } else if (message instanceof com.google.bigtable.v2.Row) {
-          result = Adapters.ROW_ADAPTER.adaptResponse((com.google.bigtable.v2.Row) message);
         } else if (message instanceof ReadModifyWriteRowResponse) {
-          result = Adapters.ROW_ADAPTER.adaptResponse(((ReadModifyWriteRowResponse) message).getRow());
+          com.google.cloud.bigtable.data.v2.models.Row row =
+              BigtableDataClientWrapper.transformResponse(((ReadModifyWriteRowResponse) message));
```

Comment:
`AsyncExecutor` still have proto model, Once updating those then we won't need to wrap message with `transformResponse()`.